.github/workflows/ci.yml

```
@@ -0,0 +1,112 @@
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
+      with:
+        file: ./coverage/lcov.info
+        flags: unittests
+        name: codecov-umbrella
+
+  build:
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)